### ISSUE TITLE
In the new NVDA settings dialog does not work numpad enter

### ISSUE THREAD

**Issue by kvark128** [nvaccess/nvda#8264]
Open any category in the settings dialog and try to press the numpad enter key, nothing happens. Enter on the main keyboard works correctly.

**Comment by ruifontes** [comment]
I confirm that issue.
And, add another issue.
1 - Open the Configurations dialog.
2 - Arrow down to voice;
3 - Tab untill Volume.
4 - Press NumPad Enter.
5 - The result will be the opening of Select Synthetizer dialog!
6 - Press Esc to close the dialog.
7 - Press the main keyboard Enter and the dialog close as it should...

**Comment by derekriemer** [comment]
@leonardder https://github.com/nvaccess/nvda/pull/7302

**Comment by Brian1Gaff** [comment]
Well, the numpad enter works on the speech dialogue to close the whole 
thing.
 I'm not sure what context you are having issues  with.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----
